cyclonedx/model/__init__.py

```diff
@@ -147,13 +226,43 @@ class ExternalReferenceType(Enum):
     LICENSE = 'license'
     MAILING_LIST = 'mailing-list'
     OTHER = 'other'
+    RELEASE_NOTES = 'release-notes'  # Only supported in >= 1.4
     SOCIAL = 'social'
     SCM = 'vcs'
     SUPPORT = 'support'
     VCS = 'vcs'
     WEBSITE = 'website'
 
 
+class XsUri:
```

Comment:
you could extend the string class .... but this might lead to a lot of possible overrides needed, so a small class is pretty fine.